src/components/Hero.tsx

```diff
@@ -0,0 +1,82 @@
+import { Box, Typography, Grid, useMediaQuery } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import HeroImage from '../../public/hero-img.png';
+import theme from '../theme';
+import { LandingPageResponse } from '../utils/types';
+
+import GradientBorderDivider from './GradientBorderDivider';
+
+interface HeroProps {
+  data: LandingPageResponse;
+}
+
+const Hero: React.FC<HeroProps> = ({ data }) => {
+  const { heroSection } = data;
+  const imageAlt = heroSection.images[0].alt;
```

Comment:
What if the alt text is different from different device sizes? I guess that should not be the case, but I guess the code has to be prepared for that.

Reply:
I don't see a use case for the image alt text / image content changing across different device sizes. In my experience, it's common to just have one image alt tag per image.